zeekpkg/package.py

```diff
@@ -57,6 +57,15 @@ def is_valid_name(name):
     return True
 
 
+def validate_aliases(metadata_dict):
+    """Return empty string if package contains valid aliases only."""
+    for a in aliases(metadata_dict):
+        if not re.match(r"^[a-zA-Z0-9][+.-_a-zA-Z0-9]*[a-zA-Z0-9]$", a):
```

Comment:
Could you add comments here why we introduce certain requirements? E.g., why do we require aliases to not start with a digit, but allow `+`/`.`/`-` in the name which would be valid file names for `@load`, but invalid `module` identifiers?

Reply:
Hmm.. This was mostly thinking "reasonably sane package", but I also realized we do not have that elsewhere.

Let me revert to simply checking for `/` and fold that into `is_valid_name()`.